### PR TITLE
V0 Under Bed WAGO Holder

### PR DESCRIPTION
**Pull request by NA411**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
QUAD 221-412 WAGO Holder

#### How was it tested? 
Printer and installed on two V0s

#### Any background context you want to provide?
Makes wiring the AC Bed easier and faster/convenient thermal fuse installation or replacement

#### Screenshots (if appropriate)
Images provided

#### Further notes
None

### CODE REVIEW

File: printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Installation
+
+Remove the front bed mount part, slide in two M3 nuts, replace the bed mount, push quad holder against the Read Bed Mounts and secure with two M3x6 BHCS or SHCS
+
+## Required Hardware
+
+1x Printed Part
+
+2x M3 Buts
```
Comment (KiloQubit) [inline printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md:16]: Change `Buts` to `Nuts`

File: printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Quad WAGO 221-412 V0 Under Bed Mount
+![Printed Part](Images/WagoHolder.png)
+![Installation](Images/Installed.jpg)
+## Overview
+
+This holds 4 221-412 WAGO Mounts for cleaner AC wiring of the heated bed on the VO. It fits nicely into the recess between the two Rear Bed Mounts. It allows for quick replacing of the fuse should it ever need to be replaced or removing the bed. The bed thermistor wiring isn't included, so I used a MOLEX connector for that.
+
+## Installation
+
+Remove the front bed mount part, slide in two M3 nuts, replace the bed mount, push quad holder against the Read Bed Mounts and secure with two M3x6 BHCS or SHCS
```
Comment (KiloQubit) [inline printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md:10]: Change `Read` to `Rear`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thank you for your mod.  Can you please

- rename the STEP file so that it doesn't have a space in the filename
- fix the mentioned spelling errors in your README
- update the top level README in `printer_mods` to add your mod to the list

Checked:

- mod is in the correct directory
- directories names contain no spaces, but one file name does
- mod includes README
- mod does NOT modify top level README
- STL is printable without support
- STL file is not corrupt
- STL file is oriented and scaled properly for printing

File: printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md (hunk 1 above)
Comment (NA411) [reply printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md]: Buts changed to Nuts

File: printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md (hunk 2 above)
Comment (NA411) [reply printer_mods/NA411/V0_Under_Bed_WAGO_Mount/README.md]: changed the d to r

**Comment by NA411** [comment]
Updated the top level README mod list with the requested information.

**Review by NA411 — COMMENTED** [review]
Updates complete of requested changes.

**Review by KiloQubit — APPROVED** [review]
Thanks for your changes, PR looks good now.

**Comment by FHeilmann** [comment]
This mod duplicates the functionality of this mod: https://github.com/VoronDesign/VoronUsers/pull/197
Do you think your mod solves a problem the other mod doesn't solve?

Bests,
-Florian

**Comment by deepfriedheroin** [comment]
I would be happy to integrate it into my repo, as an alternate version, or give my design to @NA411. I don't care about credit.

**Comment by NA411** [comment]
I agree it has the same functionality, but it mounts to the bed frame differently. I think merging the two into one repo as alternate versions is a good option

**Comment by FHeilmann** [comment]
You'll have to resolve this among yourselves and adjust the PR accordingly. Let me know if I can be of assistance.

**Comment by deepfriedheroin** [comment]
@NA411 if you want to make a pull request on my repo, go for it. don't want to take up any more of these folks time.

**Comment by NA411** [comment]
@deepfriedheroin Pull request on your Repo has been submitted.

**Comment by deepfriedheroin** [comment]
> @deepfriedheroin Pull request on your Repo has been submitted.

Apologies to maintainer for replying here. 

@NA411  I just noticed this. Will merge, in a few hours. Thank you for the contribution. 

I, actually, like your design, better than mine.

**Comment by KiloQubit** [comment]
Since these changed were added to another repo, can this pull request be closed?

**Comment by NA411** [comment]
Yes this can be closed.

**Pull request closed without merging**